datafusion/functions/src/regex/regexpreplace.rs

```diff
@@ -282,22 +282,23 @@ pub fn regexp_replace<T: OffsetSizeTrait>(args: &[ArrayRef]) -> Result<ArrayRef>
 
 fn _regexp_replace_early_abort<T: OffsetSizeTrait>(
     input_array: &GenericStringArray<T>,
+    sz: &usize,
```

Comment:
I think passing `sz` by value would be more efficient.

See: https://rust-lang.github.io/rust-clippy/master/index.html#/trivially_copy_pass_by_ref

```sh
$ cargo clippy -- -W clippy::trivially_copy_pass_by_ref
warning: this argument (8 byte) is passed by reference, but would be more efficient if passed by value (limit: 8 byte)
   --> datafusion/functions/src/regex/regexpreplace.rs:285:9
    |
285 |     sz: &usize,
    |         ^^^^^^ help: consider passing by value instead: `usize`

```